lib/api/backoff.dart

```diff
@@ -7,23 +7,38 @@ import 'dart:math';
 class BackoffMachine {
   BackoffMachine();
 
-  static const _firstDuration = Duration(milliseconds: 100);
-  static const _durationCeiling = Duration(seconds: 10);
-  static const double _base = 2;
-
   /// How many waits have completed so far.
   ///
   /// Use this to implement "give up" logic by breaking out of the loop after
   /// a threshold number of waits.
   int get waitsCompleted => _waitsCompleted;
   int _waitsCompleted = 0;
 
-  /// A future that resolves after the appropriate duration.
+  /// The bound on the duration of the first wait.
+  ///
+  /// The actual duration will vary randomly up to this value; see [wait].
+  static const firstBound = Duration(milliseconds: 100);
+
+  /// The maximum bound on the duration of each wait, even after many waits.
```

Comment:
nit: Should we mention explicitly that this, and `firstBound` are upper bounds on the first line, respectively? I feel that these are the first reads for one when looking at the `BackoffMachine` interface.